soccer/gameplay/tactics/line_up.py

```diff
@@ -8,7 +8,8 @@
 
 class LineUp(composite_behavior.CompositeBehavior):
 
-    y_start = 1.0  # sometimes we have issues if we're right in the corner, so we move it up a bit
+    DEBOUNCE_MAX = 3
```

Comment:
I added some debouncing to this play, since it was previously in the completed state when it first starts up (which makes it hard to use). It shouldn't break anything existing, I hope!